akka-actor-typed/src/main/scala/akka/actor/typed/Behavior.scala

```diff
@@ -51,7 +51,10 @@ sealed abstract class Behavior[T] { behavior ⇒
    *
    *  @param that the fallback `Behavior`
    */
-  final def orElse(that: Behavior[T]): Behavior[T] = new OrElseBehavior[T](this, that)
+  final def orElse(that: Behavior[T]): Behavior[T] = Behavior.DeferredBehavior[T] { ctx ⇒
+    val orElse = new OrElseBehavior[T](this, that)
+    orElse.start(ctx)
```

Comment:
is that indirection needed?

```
new OrElseBehavior[T](Behavior.start(this, ctx), Behavior.start(that, ctx))
```

Reply:
Doh, was staring at the Interceptor for too long